libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/ScreenLock/ScreenLockManager.swift

```diff
@@ -28,24 +28,35 @@
 import Foundation
 import SwiftUI
 
-typealias ScreenLockCallbackBlock = () -> Void
+// Callback block used to launch the app when the screen is unlocked.
+public typealias ScreenLockCallbackBlock = () -> Void
 
 @objc(SFScreenLockManager)
-class ScreenLockManager: NSObject {
-    @objc static let shared = ScreenLockManager()
+public class ScreenLockManager: NSObject {
+    @objc public static let shared = ScreenLockManager()
     
     private let kScreenLockIdentifier = "com.salesforce.security.screenlock"
     private var callbackBlock: ScreenLockCallbackBlock? = nil
     
     private override init() {}
     
-    @objc func handleAppForground() {
+    // MARK: Screen Lock Manager
+    
+    /// Locks the screen if necessary
+    @objc public func handleAppForground() {
         if readMobilePolicy() {
             lock()
+        } else {
+            unlockPostProcesing()
```

Comment:
Not having this was a really bad windowing bug but for some reason isn't necessary if `APPLICATION_EXTENSION_API_ONLY = NO` so I missed it last night.

Reply:
Typo: `unlockPostProcessing`.